recipes/ncl/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set version = "6.3.0" %}
+
+package:
+    name: ncl
+    version: {{ version }}
+
+source:
+    fn: ncl-{{ version }}.tar.gz
+    url: https://github.com/NCAR/ncl/archive/{{ version }}.tar.gz
+    sha256: d2cd8a1de1c498c1832f0765113ad58086d1d3b29bf5d2cc9b1ba60f1919951c
+
+build:
+    number: 0
+    skip: True  # [win]
```

Comment:
You can try `skip: True  # [win or osx]` for now so we can get this merged and then iterate on the feedstock to fix the `OS X` build.